### PR TITLE
new module to post metrics using finagle

### PR DESCRIPTION
**Pull request by green-creeper**

### CODE REVIEW

**Comment by koshelev** [comment]
There should not be the merge commit, only yours. please rebase it on master

File: build.sbt [hunk 1]
```diff
@@ -6,10 +6,16 @@ val Library = new {
     val circe = "0.7.1"
     val refined = "0.8.0"
     val scalaTest = "3.0.0"
+    val finagle = "6.43.0"
```
Comment (koshelev) [inline build.sbt]: these dependencies are used by single module, please move them into the `build.sbt` inside the module

File: build.sbt [hunk 2]
```diff
@@ -104,8 +119,14 @@ lazy val libats_auth = (project in file("libats-auth"))
   .settings(Publish.settings)
   .dependsOn(libats)
 
+val metrics_root = project
+  .settings(Publish.disable)
```
Comment (koshelev) [inline build.sbt]: why have you added it again?

File: libats-metrics-finagle/src/main/scala/com/advancedtelematic/metrics/FinagleMetricsSink.scala [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        case Return(value)    => promise.success(value)
+        case Throw(exception) => promise.failure(exception)
+      }
+      promise.future
+    }
+  }
+
+  def apply(settings: InfluxDbMetricsReporterSettings)(implicit system: ActorSystem,
+                                                       mat: Materializer, executionContext: ExecutionContext): Sink[String, _] = {
+    val tags             = s",service=${settings.serviceName},instance=${settings.instanceId} "
```
Comment (koshelev) [inline libats-metrics-finagle/src/main/scala/com/advancedtelematic/metrics/FinagleMetricsSink.scala]: please remove

**Comment by green-creeper** [comment]
@koshelev made all requested changes

File: build.sbt [hunk 4]
```diff
@@ -104,8 +112,12 @@ lazy val libats_auth = (project in file("libats-auth"))
   .settings(Publish.settings)
   .dependsOn(libats)
 
-lazy val libats_root = (project in file("."))
+val metrics_root = project
```
Comment (koshelev) [inline build.sbt]: Why have you added `metrics_root`?

Comment (green-creeper) [inline build.sbt]: Why? You' added this, look at e8aef3d3b92491c45337329ef066188860082662 commit

Comment (koshelev) [inline build.sbt]: And removed in https://github.com/advancedtelematic/libats/commit/46314a4c60addcce9b7c5eda25f5c78305604b32.

Comment (green-creeper) [inline build.sbt]: @koshelev fixed

**Pull request merged**